cartographer_ap/README.md

```diff
@@ -39,7 +39,7 @@ Build it with colcon build:
 ```bash
 cd ~/ros2_ws
 source /opt/ros/humble/setup.bash
-colcon build --packages-up-to ardupilot_ros ardupilot_gz_bringup
+colcon build --packages-up-to cartographer_ap ardupilot_gz_bringup
```

Comment:
Can we call it `ardupilot_cartographer_ros`? 

Then, add a metapackage, `ardupilot_ros`, that depends on `ardupilot_cartographer_ros` so we don't break out existing users? This matches the metapackage conventions like grid_map and nav2:

* https://github.com/ANYbotics/grid_map/blob/master/grid_map/package.xml
* https://github.com/ros-navigation/navigation2/blob/main/navigation2/package.xml

Reply:
Hi @Ryanf55 ! I have made the required changes and updated the commit